### PR TITLE
Add: Flag for Multi Correct

### LINKED ISSUE #6: Add a flag "multipleCorrect" to questions
A question can be either multiple correct or single correct. 

- Add a flag to indicate this. "multipleCorrect" : Boolean
- if the question is single correct, only allow one `correctAnswers` or send an error.
- Add similar check when submitting a question and quiz.

### PR DESCRIPTION
**Pull request by firefinchdev**
fixes #6

### CODE REVIEW

**Comment by firefinchdev** [comment]
@abhishek97 
Pls Review.

File: routes/api/quiz/controller.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class QuizController extends BaseController {
         model: DB.questions,
         include: {
           model: DB.choices,
-          attributes: ['id', 'title', 'positiveWeight', 'negativeWeight']
+          attributes: ['id', 'title', 'positiveWeight', 'negativeWeight', 'multiCorrect']
```
Comment (abhishek97) [inline routes/api/quiz/controller.js]: choice has no attribute: `multiCorrect`

**Review by abhishek97 — CHANGES_REQUESTED** [review]

File: routes/api/questions/controller.js [hunk 2]
```diff
@@ -29,6 +29,15 @@ class QuestionsController extends BaseController {
     const { id } = req.params
     const { correctAnswers } = req.body
 
+    let question = await this._model.findById(id, {
+      attributes: ['multiCorrect']
+    })
+
+    if((!question.multiCorrect) && (correctAnswers.length > 1)){
+      res.sendStatus(400)
```
Comment (abhishek97) [inline routes/api/questions/controller.js]: please add an appropriate error message as well.

**Review by abhishek97 — CHANGES_REQUESTED** [review]

**Comment by firefinchdev** [comment]
@abhishek97 Done. Pls review.

**Comment by firefinchdev** [comment]
@abhishek97 
Any updates?

**Pull request merged**